report.Rmd

```diff
@@ -366,17 +325,100 @@ plot_pred_int <- function(gt, model,
+
+Our estimates with the longer generation interval without adjusting for additional residual variation over time are consistent with ones from other preliminary rapid analyses using similar generation intervals (between 5-6 days), which were in the order of a 13-57% [@keeling2021], 20-60% [@kucharski2021] increase, yet lower than another estimate of 50-100% [@ferguson2021]. They are also consistent with increased secondary attack rates from contact tracing during relevant time periods, increased from 8.2% (8.0%-8.4%) for "Alpha" to 12.4% (11.7%-13.2%), corresponding with an increased reproduction number of around 50% [@phe14]. That said, other model variants yield lower estimates consistent with the data, as do shorter generation intervals, which generally lead to reproduction numbers closer to 1 and thus lower estimates of a multiplicative effect. This may be particularly relevant where the effect of the variant cause the reproduction number to cross 1.
+
+Our results should be treated with caution as several caveats apply: we assumed that S-gene positive and negative cases had the same generation interval, while a complementary hypothesis might be that the new variant shortened the generation interval, affecting our estimates [@park2021]. We assumed that the effect of tiers and lockdown applied uniformly across the country. While we did allow for a flexible regional-level behaviour through our use of UTLA level intercepts and region specifc regression splines as sensitivity analysis, there may be UTLA level, potentiallly spatial structured, variation that we did not capture in doing so. If this could explain some of the sub-regional differences in reproduction numbers, our estimate for the increased reproduction number could biased. In addition, we did not include case importation between UTLAs or cases linked to international travel which may have particularly biased initial estimates.  Lastly, we fitted the model only to the mean estimated reproduction numbers and therefore ignored uncertainty in these estimates as well as in the proportion of S-gene positives observed in every UTLA per week, which were treated as fixed point estimates. Because of this, uncertainty in our regression coefficients are underestimated, and probably considerably so. Improving our inference method to incorporate these uncertainties is a future aim of our research. Lastly, whilst "Delta" increased rapidly throughout England over the period investigated, other variants as well as, on occasion, the previously predominant "Alpha" variant may yield S-gene positives. We therefore cannot rule out that our effect includes a component not related to the new variant.
```

Comment:
Should add a limitation that we can't distinguish between advantage in transmissibility and advantage because of a greater ability to evade the immune response elicited by either prior infection or vaccination.